flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/nodes/CygnusApplication.java

```diff
@@ -363,4 +368,42 @@ private void stopSinks() {
                 
     } // AgentShutdownHook
     
+    /**
+     * Yet Another Flume Supervisor. This is a shortcut avoiding to extend the original LifecycleSupervisor class from
+     * Apache Flume, which can be be hard to do. Nevertheless, a tech debt issue has been created regarding this:
+     * https://github.com/telefonicaid/fiware-connectors/issues/354
```

Comment:
I woud add a "FIXME" mark in the comment (easy to locate all them with `grep -R FIXME`)